### PR TITLE
Using TorchANI proxy for AlanineDipeptide

### PR DESCRIPTION
**Pull request by michalkoziarski**

### CODE REVIEW

**Comment by alexhernandezgarcia** [comment]
What is the state of this PR? Waiting for review by @AlexandraVolokhova ?

**Comment by michalkoziarski** [comment]
Yup! Though after the last meeting I might as well keep working on this within this PR with sampling of baseline energy etc.

File: gflownet/envs/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-__all__ = ["base", "grid", "aptamers"]
+__all__ = ["base", "grid", "aptamers", "alaninedipeptide"]
```
Comment (AlexandraVolokhova) [inline gflownet/envs/__init__.py:1]: Just out of curiosity, could you explain what this thing is doing?

Comment (michalkoziarski) [reply gflownet/envs/__init__.py:1]: It specifies what will be imported from the module with `from ... import *`. Without it some import errors were raised.

File: gflownet/proxy/molecule.py [hunk 2]
```diff
@@ -70,36 +83,60 @@ def __init__(self, model: str = "ANI2x", use_ensemble: bool = True, **kwargs):
...
+            for elements_batch, coordinates_batch in zip(
+                torch.split(elements, self.batch_size),
+                torch.split(coordinates, self.batch_size),
+            ):
+                energies.append(self.model((elements_batch, coordinates_batch)).energies)
+            energies = torch.cat(energies).float()
+        else:
+            energies = self.model((elements, coordinates)).energies.float()
+
+        return energies / 100
```
[the remark this answers is not in the document]
Comment (michalkoziarski) [reply gflownet/proxy/molecule.py:133]: There were numerical issues without that with large betas (some rewards were `nan`s if I recall correctly).

[the remark this answers is not in the document]
Comment (AlexandraVolokhova) [reply gflownet/proxy/molecule.py:133]: I'm also thinking that maybe this scaling constant is better to be an attribute of the class, not hardcoded (but that's very minor, feel free to ignore)

**Review by AlexandraVolokhova — APPROVED** [review]

File: gflownet/proxy/molecule.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (michalkoziarski) [reply gflownet/proxy/molecule.py]: Good point, changed!

**Pull request merged**